src/components/typography/p/p.scss

```diff
@@ -19,4 +19,12 @@
     font-size: var(--typography-font-size-body-xs);
     line-height: var(--typography-line-height-body-xs);
   }
+
+  &--word-break-break-all {
+    word-break: break-all;
+  }
+
+  &--work-break-break-word {
+    word-break: break-word;
```

Comment:
I think that this one is deprecated? https://developer.mozilla.org/en-US/docs/Web/CSS/word-break#syntax